duckduckgo/duckduckgo-autofill
# l10n: pull translated strings into separate JSON files

**Reviewer:** @shakyShane 
**Asana:** https://app.asana.com/0/0/1207271815776792/f

## Description
Translations were previously hardcoded in strings.js, but continuing that pattern for more than two languages will be cumbersome. Split each language into a separate namespaced JSON file, using the single 'autofill' namespace. To prepare for future namespaces, require a namespace in string IDs passed to the translation function.

## Steps to test
This should again have no user-facing changes, sets us up for easier translations :)

1. `npm run test`
2. `npm run test:integration`
3. Use the extension + the Chrome devtools "Location" panel to force the "xa" language
4. Use the macOS app + https://github.com/duckduckgo/BrowserServicesKit/pull/808 (you may need to use the patch in that PR to force yourself into "xa" as well)
